data_steward/bq_utils.py

```diff
@@ -739,6 +739,31 @@ def create_dataset(
     return insert_result
 
 
+def get_sql_row_expr(row: dict, fields: list):
+    """
+    Get appropriate SQL expression for a row based on fields spec
+
+    :param row: csv row with untyped values
+    :param fields: bigquery fields spec
+    :return: format string for row object
+    """
+    val_exprs = []
+    # TODO refactor for all other types or use external library
+    for field_name, val in row.items():
+        field = next(filter(lambda f: f['name'] == field_name, fields), None)
+        if field is None:
+            raise InvalidOperationError(f'Unable to marshal {val}: field "{field_name}" was not found')
+        if not val and field['mode'] == 'nullable':
+            val_expr = "NULL"
+        elif field['type'] in ['string', 'date', 'timestamp']:
+            val_expr = f"'{val}'"
+        else:
+            val_expr = f"{val}"
```

Comment:
opt: existing issue, but are these input CSV files trusted in some way? What happens if someone uploads a CSV with value "; DROP TABLES" (insert actually meaningful BigQuery injection)

Reply:
@calbach this is good to think about. I have examined all usages of this function and they are all associated with our own internal use (e.g. loading lookup tables from resource files)- but I would like to prevent injection issues. Any suggestions? For now I can put a warning notice in the docstring.

Reply:
Thanks for checking. The usual way I would address this is by using a parameterized query. That's probably a big change, and I don't know whether that's something you're currently using. i.e. https://cloud.google.com/bigquery/docs/parameterized-queries, use `@param` in the query and then pass an extra parameter map when you execute the query.

Reply:
I agree, parameterized queries makes sense. That _would_ require changes to `bq_utils`- which does not currently support parameterized queries and which we are hoping to obliterate entirely now that `py3` opens us up to better bq libraries; is it ok by you that we leave this as-is and file a ticket to refactor this with parameterized queries?

Reply:
Yep, sounds good - thanks.